dronegen/gha.go

```diff
@@ -73,17 +74,57 @@ func ghaMultiBuildPipeline(setupSteps []step, ghaBuild ghaBuildType) pipeline {
 		},
 	}
 
+	setupStepNames := getStepNames(p.Steps)
+	if ghaBuild.enableParallelWorkflowRuns && len(setupSteps) >= 1 {
+		for index := range setupSteps {
+			setupSteps[index].DependsOn = append(setupSteps[index].DependsOn, setupStepNames...)
+		}
+		setupStepNames = getStepNames(setupSteps)
+	}
+
 	p.Steps = append(p.Steps, setupSteps...)
 
-	for _, workflow := range ghaBuild.workflows {
-		p.Steps = append(p.Steps, buildGHAWorkflowCallStep(workflow, checkoutPath))
+	for counter, workflow := range ghaBuild.workflows {
+		sleepTime := time.Duration(counter*10) * time.Second // 10 seconds for each workflow
```

Comment:
Why do we need a sleep here and why do sleep intervals linearly increase by 10 seconds for each subsequent workflow?